cvat-core/src/api-implementation.ts

```diff
@@ -31,7 +31,7 @@ import Webhook from './webhook';
 import { ArgumentError } from './exceptions';
 import {
     AnalyticsReportFilter, QualityConflictsFilter, QualityReportsFilter,
-    QualitySettingsFilter, SerializedAsset,
+    SettingsFilter, SerializedAsset,
```

Comment:
Why do we rename this? I think `SettingsFilter` is too broad term. 
For me, it refers to some settings inside cvat annotation view.

Reply:
The `QualitySettingsFilter` now not only applies to `QualitySettings` but also `ConsensusSettings`. 
> I think SettingsFilter is too broad term

True, I will think about a better name and then discuss.

Reply:
How about `AnalyticsSettingsFilter`?

Reply:
Ah, nice. Makes sense.